doc/visual-programming/source/widgets/classify/adaboost.rst

```diff
@@ -0,0 +1,64 @@
+AdaBoost
+========
+
+.. figure:: icons/adaboost-classification.png
+
+An ensemble meta-algorithm that combines weak learners and adapts to the 'hardness' of each training example.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+A data set. 
+
+-  **Preprocessor**
+
+Preprocessed data. 
+
+-  **Learner**
+
+A learning algorithm. 
+
+
+**Outputs**:
+
+-  **Learner**
+
+`AdaBoost <https://en.wikipedia.org/wiki/AdaBoost>`_ learning algorithm with settings as specified in the dialog.
+
+
+-  **Classifier**
+
+Trained classifier (a subtype of Classifier). The *AdaBoost classifier* signal sends data only if the learning data (signal Data) is present.
+
+Description
+-----------
+
+The **AdaBoost** (short for "Adaptive boosting") widget is a machine-learning algorithm, formulated by Yoav Freund and Robert Schapire. It can be used with other learning algorithms to boost their performance. It does so by tweaking the weak learners. 
```

Comment:
I'd link to source.
https://cseweb.ucsd.edu/~yfreund/papers/IntroToBoosting.pdf